### PR TITLE
feat: Add Safari driver options

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
## Change list

Added Safari driver options with some basic tests
 
## Types of changes

- [ ] No changes in production code.
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/main/java/io/appium/java_client/safari/options/SupportsSafariAutomaticInspectionOption.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    default T setSafariAutomaticInspection(boolean bool) {
+        return amend(SAFARI_AUTOMATIC_INSPECTION_OPTION, bool);
+    }
+
+    /**
+     * Get whether to use automatic inspection.
+     *
+     * @return true or false.
+     */
+    default Optional<Boolean> doesAafariAutomaticInspection() {
```
Comment (SrinivasanTarget) [inline src/main/java/io/appium/java_client/safari/options/SupportsSafariAutomaticInspectionOption.java:58]: `doesSafariAutomaticInspection`

File: src/main/java/io/appium/java_client/safari/options/SupportsSafariAutomaticProfilingOption.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    default T setSafariAutomaticProfiling(boolean bool) {
+        return amend(SAFARI_AUTOMATIC_PROFILING_OPTION, bool);
+    }
+
+    /**
+     * Get whether to use automatic profiling.
+     *
+     * @return true or false.
+     */
+    default Optional<Boolean> doesAafariAutomaticProfiling() {
```
Comment (SrinivasanTarget) [inline src/main/java/io/appium/java_client/safari/options/SupportsSafariAutomaticProfilingOption.java:58]: `doesSafariAutomaticProfiling`

**Review by SrinivasanTarget — CHANGES_REQUESTED** [review]

File: src/main/java/io/appium/java_client/safari/options/SupportsSafariAutomaticInspectionOption.java (hunk 1 above)
Comment (mykola-mokhnach) [reply src/main/java/io/appium/java_client/safari/options/SupportsSafariAutomaticInspectionOption.java:58]: good catch, fixed

File: src/main/java/io/appium/java_client/safari/options/SupportsSafariAutomaticProfilingOption.java (hunk 2 above)
Comment (mykola-mokhnach) [reply src/main/java/io/appium/java_client/safari/options/SupportsSafariAutomaticProfilingOption.java]: fixed

**Review by SrinivasanTarget — APPROVED** [review]

**Review by valfirst — APPROVED** [review]

**Pull request merged**